core/sbt-cloudflow/src/main/scala/cloudflow/sbt/BlueprintVerificationPlugin.scala

```diff
@@ -23,12 +23,14 @@ import sbt._
 import sbt.Keys._
 import spray.json._
 import JsonUtils._
-import cloudflow.sbt.CloudflowKeys.{ agentPaths, blueprintFile, _ }
+import cloudflow.sbt.CloudflowKeys.{ blueprintFile, _ }
 import cloudflow.blueprint._
 import cloudflow.blueprint.deployment._
 import cloudflow.blueprint.StreamletDescriptorFormat._
 
 object BlueprintVerificationPlugin extends AutoPlugin {
+  val TEMP_DIRECTORY = new File(System.getProperty("java.io.tmpdir"))
```

Comment:
Probably better to generate into target dir of sbt?

Reply:
It's a bit difficult to create in `target` as the `target` folders are different for sub-projects and the root project. We need to write from sub-project tasks (`buildAndPublish`) and read from main project task (`generateCR`) @RayRoestenburg